sdk/src/clearingHouseUser.ts

```diff
@@ -299,23 +367,39 @@ export class ClearingHouseUser {
 					bankAccount,
 					bankBalance.balanceType
 				);
-				return totalAssetValue.add(
-					tokenAmount
-						.mul(this.getOracleDataForBank(bankAccount.bankIndex).price)
-						.mul(bankAccount.initialLiabilityWeight)
-						.div(BANK_WEIGHT_PRECISION)
-						.div(MARK_PRICE_PRECISION)
+
+				let liabilityValue = tokenAmount.mul(
```

Comment:
need to scale by bank decimals e.g. https://github.com/drift-labs/protocol-v2/pull/41/files#diff-6bb1c0ea239067e0f714c8ae777ece635c2821910718ee327daca1fad1bb316aR311